SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -654,6 +693,23 @@ bool USpatialActorChannel::ReplicateSubobject(UObject* Obj, FOutBunch& Bunch, co
 	return ReplicateSubobject(Obj, RepFlags);
 }
 
+bool USpatialActorChannel::ReadyForDormancy(bool suppressLogs /*= false*/)
+{
+ 	// Check Receiver doesn't have any pending operations for this channel
+ 	if (Receiver->IsPendingOpsOnChannel(this))
```

Comment:
Does this imply we can set dormancy then change a load of state?

Reply:
This ensures that the channel that's controlling the replication of an actor that wants to go dormant has finished all replication resolution processing on that actor before allowing it to go dormant.